.golangci.yaml

```diff
@@ -0,0 +1,6 @@
+issues:
```

Comment:
The Makefile is using `staticcheck` and `go vet`, why are you adding a `golangci` configuration here, which will be ignored anyways? 🤔

Reply:
We could however switch to golangci-lint , if you want, but that opens a pandora's box of configuration options 😅 I feel we first need to agree between the three of us (@nkcr, @jbsv , @pierluca) on what a "good configuration" should be, before making the switch.